backend/package.json

```diff
@@ -20,6 +20,7 @@
     "generate-types-backend": "npx openapi-client-axios-typegen openapi.json > types/api.d.ts",
     "generate-types-account-service": "npx openapi-client-axios-typegen openapi-specs/account-service.json > types/openapi-account-service.d.ts",
     "generate-types-graph-service": "npx openapi-client-axios-typegen openapi-specs/graph-service.json > types/openapi-graph-service.d.ts",
+    "generate-types-graph-webhook": "npx @hey-api/openapi-ts -i openapi-specs/graph-service-webhook.yaml -o types/graph-service-webhook",
```

Comment:
Add this to the `generate types` script, where it runs all of the type generations.  Currently missing.

Reply:
Done